src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3088,6 +3088,23 @@
     /**
      * The {@code true} keyword.
      *
+     * <p>For example:</p>
+     * <pre>
+     * boolean a = true;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * VARIABLE_DEF -&gt; VARIABLE_DEF
+     *  |--MODIFIERS -&gt; MODIFIERS
+     *  |--TYPE -&gt; TYPE
+     *  |   `--LITERAL_BOOLEAN -&gt; boolean
+     *  |--IDENT -&gt; a
+     *  `--ASSIGN -&gt; =
+     *      `--EXPR -&gt; EXPR
+     *          `--LITERAL_TRUE -&gt; true
+     * SEMI -&gt; ;
```

Comment:
Hanging semi doesn't look good. Should we also include connection between variable_def and semi?

Reply:
Ok I'll do that.

Reply:
Done.

Reply:
Done.